lib/grppcaut.gi

```diff
@@ -971,6 +975,7 @@ local f,act,xset,stb,hom,n,m,l,i;
     Add(stb,m);
     stb:=List(stb,x->x^act); # base change
     stb:=Group(stb);
+    SetIsFinite(stb,true);
```

Comment:
Are all these `SetIsFinite` calls still necessary?

Reply:
No. SetSize (or SetIsGroupOfAutomorphismsFiniteGroup) should take care of it.